src/shared/sleep-config.c

```diff
@@ -55,6 +60,7 @@ int parse_sleep_config(const char *verb, char ***_modes, char ***_states) {
                 { "Sleep",   "HibernateState",   config_parse_strv,  0, &hibernate_state },
                 { "Sleep",   "HybridSleepMode",  config_parse_strv,  0, &hybrid_mode  },
                 { "Sleep",   "HybridSleepState", config_parse_strv,  0, &hybrid_state },
+                { "Sleep",   "ValidateHibernateGenerator", config_parse_bool, 0, &validate_hibernate_generator },
```

Comment:
That's pretty obscure name. ValidateResumeKernelParameter is too long but at least it makes clear what this controls.